lib/discovery.js

```diff
@@ -63,20 +63,25 @@ function mixinDiscovery(DB2, db2) {
 
     if (options.all && !schema) {
       sqlTables = paginateSQL('SELECT \'table\' AS "type",' +
-        ' tabname AS "name", tabschema AS "owner", property as "property"' +
+        ' TRIM(tabname) AS "name",' +
+        ' TRIM(tabschema) AS "owner",' +
+        ' TRIM(property) as "property"' +
```

Comment:
Any reason why sometimes `AS` is uppercase and the other ones are lowercase `as`?